lib/fastlane/helper/git_helper.rb

```diff
@@ -1,9 +1,12 @@
 module Fastlane
   module Actions
-    def self.git_log_between(pretty_format, from, to, include_merges)
+    GIT_MERGE_COMMIT_FILTERING_OPTIONS = [:include_merges, :exclude_merges, :only_include_merges].freeze
```

Comment:
You're right, no reason to build the flexibility for flexibility sake. I'm cool with keeping it like this.